utils/src/ooinstall/cli_installer.py

```diff
@@ -72,7 +72,7 @@ def delete_hosts(hosts):
                 click.echo("\"{}\" doesn't coorespond to any valid input.".format(del_idx))
     return hosts, None
 
-def collect_hosts(oo_cfg, masters_set=False, print_summary=True):
+def collect_hosts(oo_cfg, existing_env=False, masters_set=False, print_summary=True):
```

Comment:
@detiber That's a good idea and not too hard to do instead. I'll start hacking on doing it that way.